xlml/utils/xpk.py

```diff
@@ -63,18 +64,24 @@ def run_workload(
       multi_keyword = "num-nodes"
     else:
       multi_keyword = "num-slices"
+
+    workload_create_cmd = (
+        f"python {tmpdir}/xpk/xpk.py workload create"
+        f" --cluster={cluster_name} --workload={workload_id}"
+        f" --command='{run_cmds}' --device-type={accelerator_type}"
+        f" --{multi_keyword}={num_slices} --docker-image={docker_image}"
+        f" --project={cluster_project} --zone={zone}"
+        f" --env {metric_config.SshEnvVars.GCS_OUTPUT.name}={gcs_path}"
+        " --restart-on-user-code-failure"
+    )
+    if use_vertex_tensorboard:
+      workload_create_cmd += " --use-vertex-tensorboard"
+
     cmds = (
         "set -xue",
         f"git clone https://github.com/google/xpk {tmpdir}/xpk",
-        (
-            f"python {tmpdir}/xpk/xpk.py workload create"
-            f" --cluster={cluster_name} --workload={workload_id}"
-            f" --command='{run_cmds}' --device-type={accelerator_type}"
-            f" --{multi_keyword}={num_slices} --docker-image={docker_image}"
-            f" --project={cluster_project} --zone={zone}"
-            f" --env {metric_config.SshEnvVars.GCS_OUTPUT.name}={gcs_path}"
-            " --restart-on-user-code-failure"
-        ),
+        "pip install cloud-accelerator-diagnostics",
```

Comment:
Chatted offline - this is a dependency of `xpk` to create Vertex tensorboards. Since the package will update frequently, we'll keep a direct `pip install -U` here. Thanks @SurbhiJainUSC!